docs/internal-documentation/maintenance/IssueLabels.md

```diff
@@ -3,54 +3,36 @@
-Area     | A-Documentation  | green  | all                | Documentation not covered by other areas
-Area     | A-Installation   | green  | all but "main"     | Installation instructions of Zonemaster for users
-Area     | A-Logging        | green  | Backend            | Documentation or implementation of logging system in Zonemaster-Backend
-Area     | A-OSintegration  | green  | Backend            | Code or documentation of integration of Zonemaster into the OS
-Area     | A-RCPAPI         | green  | Backend            | Documentation or implementation of the RPCAPI in Zonemaster-Backend
-Area     | A-TestAgent      | green  | Backend            | Documentation or implementation of the Test Agent in Zonemaster-Backend
-Area     | A-TestCase       | green  | "main" or Engine   | Test case specification or implementation of test case
-Area     | A-Translation    | green  | all                | Documentation of, implementation of or actual translation of text
-Area     | A-Travis         | green  | all but "main"     | Travis test tool in Github
-Area     | A-UnitTests      | green  | all but "main"     | Documentation or implementation of unit tests
-Priority | P-High           | red    | all                | Issue to be solved before other
-Status   | S-InProgress     | yellow | all                | Work in progress, but no proposal or pull request exists
-Status   | S-ProposalExists | yellow | all                | Work in progress and a proposal (PR) exists
-Status   | S-Stalled        | yellow | all                | Work has been stopped or stalled or is waiting for some other issue to resolved before continuing
-Type     | T-Bug            | red    | all                | Bug in software or error in test case description
-Type     | T-Feature        | blue   | all                | New feature in software or test case description
-Type     | T-Performance    | blue   | all but "main"     | Performance in software
-Type     | T-Project        | blue   | "main"             | Project related issue
-Type     | T-Question       | blue   | all                | Incoming question
-Type     | T-Stability      | blue   | all but "main"     | Stability in software
-Type     | T-Usability      | blue   | all but "main"     | Usability of software
+Area     | A-Documentation  | green  | all                | Documentation only.
+Area     | A-FA-Backend-conf| green  | Backend            | Focus Area Improving backend configuration handling.
+Area     | A-FA-CDS-CDNSKEY | green  | "main" or Engine   | Focus Area Adding test cases for CDS and CDNSKEY.
+Area     | A-FA-MethodNT    | green  | "main" or Engine   | Focus Area Implementing and migrating to MethodNT for test cases.
+Area     | A-FA-Basic04     | green  | "main" or Engine   | Focus Area Implementing Basic04 and adjusting other test cases to it.
+Area     | A-FA-IDN-ASCII   | green  | Backend or GUI     | Focus Area Moving U-label-to-A-label conversion to GUI and zmtest and away from RPCAPI.
```

Comment:
IMHO these labels are a bit long and chopped up. Maybe they could be called F-BackendConf, F-CDS and F-IDN or something? I believe it would be easier on the eye to parse a horde of them that way.

Reply:
The reason that all Area labels start with "A-" is to make sure they are sorted together. Would it be better with "A-Backend-conf-FA"?

Reply:
Updated FA labels pushed.